src/main/dsl/Node.kt

```diff
@@ -1,5 +1,9 @@
 package dsl
 
+@DslMarker
+annotation class NodeMarker
+
+@NodeMarker
 abstract class Node {
     abstract fun render(builder: StringBuilder)
```

Comment:
I removed Node in my local branch, so you might want to check that out and see if our code is still consistent.